pycqed/analysis_v2/base_analysis.py

```diff
@@ -339,7 +354,7 @@ def get_param_value(self, param_name, default_value=None, metadata_index=0):
         elif isinstance(self.metadata, (list, tuple)) and \
                 len(self.metadata) != 0:
             return self.options_dict.get(param_name,
-                self.metadata[metadata_index].get(param_name, default_value))
+                                         self.metadata[metadata_index].get(param_name, default_value))
```

Comment:
Line too long. You should add a line break in the previous line before param_name. Then, you need less indent.